docs/sql-programming-guide.md

```diff
@@ -479,6 +481,26 @@ source type can be converted into other types using this syntax.
 </div>
 </div>
 
+To load a CSV file you can use:
+
+<div class="codetabs">
+<div data-lang="scala"  markdown="1">
+{% include_example manual_load_options_csv scala/org/apache/spark/examples/sql/SQLDataSourceExample.scala %}
+</div>
+
+<div data-lang="java"  markdown="1">
+{% include_example manual_load_options_csv java/org/apache/spark/examples/sql/JavaSQLDataSourceExample.java %}
+</div>
+
+<div data-lang="python"  markdown="1">
+{% include_example manual_load_options_csv python/sql/datasource.py %}
+</div>
+
+<div data-lang="r"  markdown="1">
+{% include_example manual_load_options_csv r/RSparkSQLExample.R %}
+
+</div>
+</div>
 ### Run SQL on files directly
```

Comment:
Yup, that's okay. BTW, I initially what I meant in https://github.com/apache/spark/pull/19429#discussion_r143932389 was a newline between `</div>` and `### Run ..` (not `...ample.R %}` and `</div>`. This breaks rendering:

<img src="https://user-images.githubusercontent.com/6477701/31481516-cd9ddb80-af5e-11e7-970b-d2c279f025d4.png" width="200" />


Let's don't forget to fix this up before the release if the followup couldn't be made ahead.

Reply:
@HyukjinKwon  should I add a new line between line 503 and 504 ? 
For example : 
```
{% include_example generic_load_save_functions r/RSparkSQLExample.R %}

</div>
</div>

### Manually Specifying Options
```

Reply:
Yup, a newline between 503 and 504.